### ISSUE TITLE
9.0 Summary of Proposed System (PHASE 3)

### ISSUE THREAD

**Issue by nikbalqis** [ar1sha55/SEC2613-02-Group-4-SAD-Project#34]

**Comment by tjasmiine** [comment]
# 9.0 Summary of Proposed System

This proposed system consists of `6 processes` in the Level 0 of Physical DFD, in which the processes are:
<pre>
1. Register Account (Manual)
2. Login into Account (Manual)
3. Pass Request to Admin (Automated)
4. Matching Profile (Automated)
5. Track Student Thesis Progress (Manual)
6. Correction, Review and Submit Thesis (Manual)
</pre>

Each process has its own input and output and benefits to the system, and this is demonstrated by doing the process specification, CRUD Matrix, event response table, structure chart and input/output design of each of them.

The gist of this system is that students can register then login into the system, then request a supervisor by browsing through the system. Then, the system will do a profile matching with that particular supervisor which is an automated process. Consequently, the system will generate an accept/decline message to notify the student of their request success. Once assigned to a supervisor, the student may merge their calendar and start to track their thesis progress.

In order to track their thesis progress, they may add the progress on the calendar where both student and supervisor will be able to monitor the progress. After appointments with the supervisor, students can submit their thesis draft and the supervisor will review and correct their thesis if needed. Students are able to upload their thesis draft repeatedly until both student and supervisor are satisfied with the outcome.

This system is the contribution of our 4 group members, who worked together by delegating our tasks and discussing it to ensure the best outcome. This system is also majorly contributed by Dr. Muhammad Aliif, our lecturer who constantly gives feedback on our progress and guides us through the process of finishing this phase.

Though we find this proposed system already beneficial, we believe that it can be improved by enhancing user interface to improve user’s experience, linking this website to their respective institute for additional features like applying for scholarship from their institute and expanding the website to mobile phone as well to ensure better integration of the system.

In conclusion, this project has been a success due to the contribution by every member and guidance from our lecturer. Our hope for this proposed system is that it can help all postgraduate students who are in the process of finding a supervisor as we are aware that it can be a hard phase for them. We also hope that this proposed system will be implemented and commercialized all across the country.
